### PR TITLE
feature: support TensorFlow 2.2 Python 3.7

### PR DESCRIPTION
**Pull request by ChuyangDeng**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pipemode_op/RecordReader/RecordIOReader.cpp [hunk 1]
```diff
@@ -64,8 +66,8 @@ bool RecordIOReader::ReadRecord(std::string* storage) {
         std::size_t expected_size = GetRecordSize(header);
         std::size_t padded_expected_size = GetPaddedSize(expected_size);
         total_record_size += expected_size;
-        storage->resize(total_record_size);
-        Read(&(storage->at(total_record_size - expected_size)), expected_size);
+        storage->resize_uninitialized(total_record_size);
+        Read(&((*storage)[total_record_size - expected_size]), expected_size);
```
Comment (yuanzhua) [inline src/pipemode_op/RecordReader/RecordIOReader.cpp:70]: Is at() function deprecated?

File: src/pipemode_op/RecordReader/TextLineRecordReader.cpp [hunk 2]
```diff
@@ -55,7 +57,6 @@ bool TextLineRecordReader::ReadRecord(std::string* data) {
             if (data->size() == 0) {
                 return false;
             } else {
-                data->shrink_to_fit();
```
Comment (yuanzhua) [inline src/pipemode_op/RecordReader/TextLineRecordReader.cpp:58]: why remove this call? Is it deprecated or no longer required?

File: src/pipemode_op/RecordReader/RecordIOReader.cpp (hunk 1 above)
Comment (ChuyangDeng) [reply src/pipemode_op/RecordReader/RecordIOReader.cpp:70]: tensorflow::tstring does not implement at()

File: src/pipemode_op/RecordReader/TextLineRecordReader.cpp (hunk 2 above)
Comment (ChuyangDeng) [reply src/pipemode_op/RecordReader/TextLineRecordReader.cpp:58]: tensorflow::tstring does not implement shrink_to_fit(), re-implemented as resize to it's string size

**Pull request merged**